src/carplay.cs

```diff
@@ -1907,4 +1925,22 @@ interface CPInformationRatingItem
 		[NullAllowed, Export ("maximumRating")]
 		NSNumber MaximumRating { get; }
 	}
+
+	[NoWatch, NoTV, NoMac, iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof(NSObject))]
+	interface CPAssistantCellConfiguration : NSSecureCoding
```

Comment:
-> `[DisableDefaultCtor]`
there's no way to change the default, empty/null, value if `init` is used - so it's unusable